### PR TITLE
[sflow] Fix traceback seen for show sflow interface

### PR DESCRIPTION
**Pull request by GarrickHe**
* Fix traceback seen for 'show sflow interface' command

Signed-off-by: Garrick He <garrick_he@dell.com>

**- What I did**
Fixed a traceback seen when `show sflow interface` is executed.

**- How I did it**
Updated the script to follow Python3 conventions.

**- How to verify it**
Retested the command to ensure no traceback is seen.

### CODE REVIEW

**Comment by GarrickHe** [comment]
This PR fixes this issue:
https://github.com/Azure/sonic-buildimage/issues/6078

File: show/main.py [hunk 1]
```diff
@@ -1648,11 +1648,11 @@ def show_sflow_interface(config_db):
     for pname in natsorted(list(port_tbl.keys())):
         intf_key = 'SFLOW_SESSION_TABLE:' + pname
         sess_info = sess_db.get_all(sess_db.APPL_DB, intf_key)
-        if sess_info is None:
+        if sess_info is None or sess_info.get('admin_state') is None:
```
Comment (prsunny) [inline show/main.py:1651]: why check only 'admin_state'? how about 'sample_rate' being None?

**Comment by prsunny** [comment]
@padmanarayana to review

File: show/main.py (hunk 1 above)
Comment (GarrickHe) [reply show/main.py:1651]: will update.

**Review by padmanarayana — DISMISSED** [review]

**Comment by GarrickHe** [comment]
@prsunny - Test failures doesn't appear to be sFlow related.

**Comment by prsunny** [comment]
retest this please

**Review by prsunny — DISMISSED** [review]

**Comment by GarrickHe** [comment]
retest this please

**Comment by lguohan** [comment]
can you add unit test case?

**Comment by GarrickHe** [comment]
@lguohan  - No problem. Will add test-case tomorrow. Moving this PR to draft for now.

**Comment by GarrickHe** [comment]
@prsunny @lguohan  @padmanarayana  - I added the required test-cases. I ran it with and without the fix to ensure this issue is covered.

File: tests/sflow_test.py [hunk 2]
```diff
@@ -261,6 +261,35 @@ def test_config_sflow_intf_sample_rate(self):
 
         return
 
+    def test_config_disable_all_intf(self):
```
Comment (prsunny) [inline tests/sflow_test.py:264]: I dont see the test covering the app db entries or the 'show' command. ??

Comment (GarrickHe) [reply tests/sflow_test.py:264]: I added the fault entries into appl_db.json and the `test_show_intf()` on line 55 will ensure we don't traceback due to the bad entries. 

The two new test I added `test_config_enable_all_intf` and `test_config_disable_all_intf` merely ensures the proper configDB entries get added when the user runs those commands.

**Review by prsunny — APPROVED** [review]

**Comment by GarrickHe** [comment]
@prsunny  @lguohan  - This is ready. let me know if there is anything else you need on my end. Thanks.

**Pull request merged**